src/routes/namespaces/[namespace]/workflows/_workflows-summary-row.svelte

```diff
@@ -56,15 +56,15 @@
   }
 
   tr {
-    @apply bg-gray-50;
+    @apply border-2;
   }
 
   tr:hover {
-    @apply bg-gray-100;
+    background-color: #e6fffa;
```

Comment:
I'd just pick the closest Tailwind color.

Reply:
Noted